### ISSUE TITLE
Canceling Twitter OAuth Flow gives error

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis-rest#509]
When canceling the Twitter OAuth Flow via the "Cancel" button on the Twitter side, after the redirect the following error appears:

```json
{"errorCode":404,"userMsg":"Please check your request data","developerMsg":"Entity Not Found Exception Unable to finish OAuth authorization via callback, cannot find the OAuth flow state this callback request relates to"}
```

**Comment by zregvart** [comment]
see #464 and #514

**Comment by zregvart** [comment]
This should have been fixed with #577
